.github/workflows/ci-unwelcome-words.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
```

Comment:
could you please discuss @Xanclry about using common workflow from .github repository
https://github.com/th2-net/.github/pull/7/files

Reply:
@Xanclry could you please help? Because I don't really understand what should be done here. Should I add some of the workflows from the PR above?